lib/cdo/shared_constants.rb

```diff
@@ -878,6 +878,12 @@ module SharedConstants
     {value: "other", label: "Other", category: 'other'}
   ].freeze
 
+  RESOURCE_EMBEDDABILITY_OPTIONS = {
+    EMBED_AND_RESOURCE_DROPDOWN: {value: "embed_and_resource_dropdown", label: "Both embedding and resource dropdown"},
+    EMBED_ONLY: {value: "embed_only", label: "Embedding only"},
+    RESOURCE_DROPDOWN_ONLY: {value: "resource_dropdown_only", label: "Resource dropdown only"}
```

Comment:
I think the wording here could be clearer about what "embedding" actually is.
```suggestion
    EMBED_AND_RESOURCE_DROPDOWN: {value: "embed_and_resource_dropdown", label: "AI TA knowledge base and resource dropdown"},
    EMBED_ONLY: {value: "embed_only", label: "AI TA knowledge base only"},
    RESOURCE_DROPDOWN_ONLY: {value: "resource_dropdown_only", label: "Resource dropdown only"}
```

Reply:
Good call! Will do!